src/transformers/models/detr/modeling_detr.py

```diff
@@ -23,6 +23,8 @@
 import torch
 from torch import Tensor, nn
 
+from transformers import AutoBackbone
```

Comment:
This should be imported with a relative import from its module (as all other objects below).